src/desktop-lite/install.sh

```diff
@@ -198,6 +197,15 @@ fi
 # Install X11, fluxbox and VS Code dependencies
 check_packages ${package_list}
 
+# if Ubuntu-24.04 (noble numbat) found, then install libasound2-dev else libasound2
```

Comment:
Can we add a note that this is temporary and we will swtich to `libasound2` once noble is available.

Also, add https://packages.ubuntu.com/noble/libasound2  link in the comment

Reply:
done